lib/decision/polling/polling.dart

```diff
@@ -0,0 +1,25 @@
+import 'dart:async';
+
+class Polling {
+  final int intervalTimePolling;
+
+  Function() getScript;
+  late Timer timer;
+
+  Polling(this.intervalTimePolling, this.getScript);
+
+  start() {
+    getScript().whenComplete(() {
```

Comment:
Ah ok, you execute the script first on start. I guess that resolves my other comments on the file being empty in the first call to getCampaign.
But it could still be the case if the dev calls `getCampaigns` before the first call to the bucketing file is done

Reply:
Yes the use cas is possible, if the dev fetch immediately before the first polling is done. 
I remove the the exception , the user will see the log message "**Flagship, Failed to synchronize**"